src/output.c

```diff
@@ -122,6 +143,24 @@ static void *outputThread(void *data) {
       inPic.i_pts = (int64_t)(duration * i);
       x264_encoder_encode(x264, &nals, &nalCount, &inPic, &outPic);
       outputNals(&track, nals, nalCount, duration);
+
+      ts += (OUTPUT_TIMEBASE / output->config->framerate) * output->config->audioChannels;
+      while (ats < ts) {
+         uint8_t buf[AAC_OUTPUT_BUFFER_SIZE];
+         int numBytes = 0;
+         int numSamples = 0;
+         rsAudioEncoderEncode(&audioenc, output->rawSamples, buf, &numBytes, &numSamples);
+         output->rawSamples += audioenc.frameSize;
+         samplesCount += numSamples;
+         ats = (int64_t)samplesCount * OUTPUT_TIMEBASE / output->config->audioSamplerate;
+         if (MP4E_STATUS_OK !=
+             MP4E_put_sample(muxer, audio_track_id, buf, numBytes,
+                             (numSamples / output->config->audioChannels) *
+                                 OUTPUT_TIMEBASE / output->config->audioSamplerate,
+                             MP4E_SAMPLE_RANDOM_ACCESS)) {
+            rsError("MP4E_put_sample failed\n");
+         }
+      }
    }
    while (x264_encoder_delayed_frames(x264) > 0) {
```

Comment:
```suggestion
   }

   while (x264_encoder_delayed_frames(x264) > 0) {
```
Despite it being my code I feel like the space is needed now ;p